subprojects/gradle/instrumentation-tests-test-fixtures/src/main/kotlin/com/avito/instrumentation/reservation/client/kubernetes/FakeDeploymentNameGenerator.kt

```diff
@@ -0,0 +1,11 @@
+package com.avito.instrumentation.reservation.client.kubernetes
+
+import com.avito.utils.gradle.toValidKubernetesName
+
+class FakeDeploymentNameGenerator(
+    private val buildId: String
```

Comment:
Do we need to know a real `buildId`? 
It seems the random value is enough.

Reply:
yep, fixed